src/dev_deps.ts

```diff
@@ -4,3 +4,5 @@ export {
   assertStringIncludes,
 } from "https://deno.land/std@0.138.0/testing/asserts.ts";
 export * from "https://deno.land/x/mock_fetch@0.3.0/mod.ts";
+
+export * as mockFile from "https://deno.land/x/mock_file@v1.0.1/mod.ts";
```

Comment:
New mocking utility for files. Shockingly easy and intuitive to use.

Reply:
Except for "unmounting" a previously virtually-mounted file. Not sure how to do that just yet. 🤔